### DISCUSSION TITLE
Installation on Replit

### DISCUSSION THREAD

**Discussion by shaggy2626** [danny-avila/LibreChat#1104] [Troubleshooting] [answered]
I'm not great with code and was wondering if there was a simple way to install Libre on Replit. I created a new replit and imported the git but wasn't sure what to do next, I just want to be able to use Libre for my own personal use. any help would be appreciated, thank you.

**Comment by madonchik123** [comment]
You can't install this on replit because replit doesn't support docker I've recommend using render if you know how to use it

**Comment by danny-avila** [comment, marked as answer]
There are several installation guides here: https://docs.librechat.ai/install/docker_compose_install.html

We also have many different deployment guides: https://docs.librechat.ai/deployment/digitalocean.html

**Reply by gitGunnl** [reply]
Hi. Do you know if there is any way to run LibraChat on Replit now?

I don't seem to see any way in the documentation, but have you seen any example of it running on Replit?
